tests/test_items.py

```diff
@@ -100,6 +100,31 @@ def test_patch_item__success(self):
         assert json_item['price'] == TEST_ITEM['price']
         assert json_item['description'] == TEST_ITEM['description']
         assert json_item['item_id'] == item.item_id
+        resp_item = Item.get(item_id=item.item_id).json()
+        assert resp_item['name'] != 'new-name'
```

Comment:
il test è scritto un po' al contrario di come dovrebbe essere. Se in una PATCH invii solamente il campo `name` allora il server deve modificare solo quello. La risposta quindi del server e il contenuto del database dovrebbe essere quindi 'new-name' per il campo `name` e il vecchio valore per gli altri campi. Qua invece stai facendo:
```
assert json_item['name'] != 'new-name'
```